.github/workflows/ci.yml

```diff
@@ -94,6 +94,10 @@ jobs:
     - name: Run tests
       run: |
         ./y.sh test --release --clean --build-sysroot ${{ matrix.commands }}
+      
+    - name: Run y.sh cargo build
+      run: |
+        ./y.sh cargo build    
```

Comment:
This will attempt to compile `rustc_codegen_gcc` itself which is a bit complicated to build.

Instead, I would suggest that you add a "Hello, world!" project in the `tests` directory and build this new simple project instead.